homeassistant/components/aladdin_connect/__init__.py

```diff
@@ -24,9 +24,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     try:
         if not await hass.async_add_executor_job(acc.login):
             raise ConfigEntryAuthFailed("Incorrect Password")
-    except (TypeError, KeyError, NameError, ValueError) as ex:
-        _LOGGER.error("%s", ex)
-        raise ConfigEntryNotReady from ex
+    except ValueError as ex:
```

Comment:
When can login raise ValueError?

https://github.com/shoejosh/aladdin-connect/blob/a351c6ce6c407e255a4954cd05b3c2fdb367f99d/aladdin_connect/__init__.py#L56-L57

https://github.com/shoejosh/aladdin-connect/blob/a351c6ce6c407e255a4954cd05b3c2fdb367f99d/aladdin_connect/session_manager.py#L35-L60

Reply:
Well... never. I guess I missed the fact it was not re-raised.